.env.example

```diff
@@ -3,3 +3,4 @@ REACT_APP_OAUTH_CLIENT_ID='ENTER CLIENT ID'
 GOOGLE_APPLICATION_CREDENTIALS='PATH TO YOUR SERVICE ACCOUNT JSON FILE'
 GS_BUCKET_NAME='GS BUCKET NAME'
 SIGNING_KEY='YOUR SIGNING KEY'
+DJANGO_ENV='DEVELOPMENT'
```

Comment:
We can check that by just togggling value of DEBUG
if DEBUG = True - Dev
else Deployment